docker/colony-cdapp-dev-env-orchestration.yaml

```diff
@@ -88,7 +88,7 @@ services:
     image: colony-cdapp-dev-env/base:latest
     volumes:
       - 'amplify-mock-data:/mock-data'
-    command: "npx --yes http-server --cors --port 3006 /mock-data/colonyNetworkArtifacts"
+    command: 'npx --yes http-server --cors --port 3006 /mock-data/colonyNetworkArtifacts'
```

Comment:
:eyes: ?

Reply:
This is my linter automatically formatting it with single quotes. This yaml file is basically using single quotes for the entire file and this is the only string with double quotes. I did the same thing here: https://github.com/JoinColony/colonyCDapp/pull/2519#discussion_r1642735998